packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.base.tsx

```diff
@@ -203,8 +203,10 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, any> {
    * @param props Props to validate
    */
   private _validateProps(props: IBreadcrumbProps): void {
-    const { items, overflowIndex } = props;
-    if (overflowIndex! < 0 || overflowIndex! > items.length - 1) {
+    const { maxDisplayedItems, overflowIndex, items } = props;
+    if (overflowIndex! < 0 ||
+      maxDisplayedItems! > 1 && overflowIndex! > maxDisplayedItems! - 1 ||
```

Comment:
Should overflowIndex be compared against maxDisplayedItems when maxDisplayed items is 1?

Reply:
I think this is enough sanity checking for the moment. The component doesn't actually break if you specify values out-of-bounds.